install

```diff
@@ -135,28 +141,40 @@ do_install()
   rosinstall_generator ${ROS_CONFIGURATION} ${ROS_EXTRA_PACKAGES} --rosdistro ${ROS_DISTRO} --deps --tar > ${WS}.rosinstall
   wstool init src
   pushd src
-    # Avoid downloading opencv3; we already installed it from homebrew.
     wstool merge file://$(pwd)/../${WS}.rosinstall
-    wstool remove opencv3
     wstool update -j8
 
-    # See: https://github.com/ros/catkin/pull/784
-    if [ -d catkin ]; then
-      curl https://raw.githubusercontent.com/ros/catkin/8a47f4eceb4954beb4a5b38b50793d0bbe2c96cf/cmake/catkinConfig.cmake.in > catkin/cmake/catkinConfig.cmake.in
+    # Pending release https://github.com/ros/bond_core/pull/37
+    if [ -d bond_core ]; then
+      curl https://raw.githubusercontent.com/ros/bond_core/1240bce50ca1538cdad07dc472d31797257e0851/bondcpp/src/bond.cpp > bond_core/bondcpp/src/bond.cpp
```

Comment:
Just a note: Maybe applying the patch instead of taking the files as is would be slightly more future proof, in case of a new release with other unrelated changes but this PR not merged. It also means just one line per PR and you get the latest version when the PR is updated.
```
curl https://patch-diff.githubusercontent.com/raw/ros/bond_core/pull/37.patch | patch -p1 --dir bond_core
```

Same for the other fixes from PR.

Reply:
PS: overall I needed the following changes (from PRs) on Sierra for melodic:

```
    # See: https://github.com/ros/catkin/pull/784
    if [ -d catkin ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros/catkin/pull/784.patch | patch -p1 --dir catkin
    fi

    # See: https://github.com/ros/bond_core/pull/37
    if [ -d bond_core ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros/bond_core/pull/37.patch | patch -p1 --dir bond_core
    fi

    # See: https://github.com/ros-perception/image_common/pull/85
    if [ -d image_common ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros-perception/image_common/pull/85.patch | patch -p1 --dir image_common
    fi

    # See: https://github.com/ros-perception/vision_opencv/pull/239
    if [ -d vision_opencv ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros-perception/vision_opencv/pull/239.patch | patch -p1 --dir vision_opencv
    fi

    # See: https://github.com/ros/actionlib/pull/111
    # See: https://github.com/ros/actionlib/pull/105
    if [ -d actionlib ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros/actionlib/pull/111.patch | patch -p1 --dir actionlib
        curl https://patch-diff.githubusercontent.com/raw/ros/actionlib/pull/105.patch | patch -p1 --dir actionlib
    fi

    # See: https://github.com/ros-perception/laser_assembler/pull/16
    if [ -d laser_assembler ]; then
        curl https://patch-diff.githubusercontent.com/raw/ros-perception/laser_assembler/pull/16.patch | patch -p1 --dir laser_assembler
    fi
```